did/peer/peer_test.go

```diff
@@ -1,13 +1,27 @@
-package did
+package peer
 
 import (
 	"context"
 	"testing"
 
 	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+
 	"github.com/stretchr/testify/assert"
 )
 
+func TestEncodePublicKeyWithKeyMultiCodecType(t *testing.T) {
+	// unsupported type
+	_, err := encodePublicKeyWithKeyMultiCodecType(crypto.KeyType("unsupported"), nil)
+	assert.Error(t, err)
+	assert.Contains(t, err.Error(), "not a supported key type")
+
+	// bad public key
+	_, err = encodePublicKeyWithKeyMultiCodecType(crypto.Ed25519, nil)
+	assert.Error(t, err)
+	assert.Contains(t, err.Error(), "unknown public key type; could not convert to bytes")
```

Comment:
Similar comment to the above. Though I see this was only moved. Up to you.

Reply:
done